docs/pages/zencode-scenarios-eddsa.md

```diff
@@ -0,0 +1,66 @@
+
+# EdDSA
+
+[EdDSA](https://en.wikipedia.org/wiki/EdDSA)(*Edwards-curve Digital Signature Algorithm*) is a variant of Schnorr signature algorithm. The main change apported in this scheme is the employment of twisted Edwards-curves that result in having higher perfomance with respect to other digital signature algorithms, wihtout sacrificing security. Moreover this scheme resolve also another major issue emerged in the other signature schemes: it **does not need a criptographically secure random number generator** to select a random parameter.
```

Comment:
easier to say signature here instead of parameter.

well done adding emphasis.